libethereum/Client.cpp

```diff
@@ -631,7 +631,8 @@ void Client::rejigSealing()
 			if (wouldSeal())
 			{
 				sealEngine()->onSealGenerated([=](bytes const& header){
-					this->submitSealed(header);
+					if (!this->submitSealed(header))
```

Comment:
wont this throw an exception in Block.cpp ?
https://github.com/ethereum/libethereum/blob/develop/libethereum/Block.cpp#L302